python/sqlalchemy/src/yuga_demo.py

```diff
@@ -0,0 +1,283 @@
+#!/usr/bin/env python
```

Comment:
Can we rename this file to something like `rest-service.py`

Reply:
Done -- good suggestion